tests/spread/plugins/v1/ruby/snaps/ruby-bundle-install/snap/snapcraft.yaml

```diff
@@ -17,3 +16,6 @@ parts:
     plugin: ruby
     source: .
     use-bundler: true
+    ruby-version: "2.6.10" # bundler requires ruby >= 2.6.0
```

Comment:
I'm cautious to make changes to the v1 plugin itself, due to the fact that it's been stable for a long time and that I don't know much about Ruby.

I am considering updating the documentation to mention the `BUNDLE_GEMFILE` environment variable.  Or it could become a parameter in the snapcraft.yaml (i.e. `ruby-gemfile-location: <path-to-gemfile>`).

Reply:
For now I think this is a good minimal change to get this spread test fixed. However, if it works on Ruby 2.7 that'd probably be reasonable since it should be fully backwards compatible.

In theory Ruby 3 is backwards compatible too, but we should probably make a separate project for bringing our ruby deps up to the latest.